config.py

```diff
@@ -7,3 +7,6 @@
 SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(basedir, 'app.db')
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
 SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+per_page = 10
+pagination_framework = 'foundation'
```

Comment:
Minor comments, feel free to ignore: config variables are in capitals by convention
Also another convention: have a newline at the end of python files